test/unit/lodash.spec.js

```diff
@@ -0,0 +1,522 @@
+                    expect(typeof params[1]).to.eql('string');
+                });
+
+                it('should return min value from Object', function () {
+                    expect(_.min.apply(_, params)).to.eql({ Name: 'Jack', Gold: 1500 });
+                });
+            });
+
+            describe('minBy - String', function () {
+
+                var params;
+
+                beforeEach(function () {
+                    params = datasets.minBy4();
+                });
+                
+                it('should verify params', function () {
+                    expect(params).to.have.length(1);
+                    expect(typeof params[0]).to.eql('string');
+                });
+                
+                it('should return min value from string', function () {
+                    expect(_.min.apply(_, params)).to.eql('e');
+                });
+            });
+        });
+
+        describe('multiply', function () {
+
+            var val1;
+            var val2;
+            var result1;
+            var params = datasets.multiply();
+
+            beforeEach(function () {
+                val1 = Math.round(Math.random() * 10);
+                val2 = Math.round(Math.random() * 10);
+                result1  = val1 * val2;
+            });
+
+            it('should multiply two random number', function () {
+                expect(params).to.have.length(2);
+                expect(_.multiply.apply(_, datasets.multiply(val1, val2))).to.eql(result1);
+            });
+        });
+
+        describe('round', function () {
+
+            var params;
+            var sqrt;
+
+            beforeEach(function () {
+                params = datasets.round();
+                sqrt = params[0];
+            });
+
+            it('should verify', function () {
+                expect(sqrt instanceof Function).to.eql(true);
+                expect(params).to.have.length(2);
+            });
+
+            it('should return sqrt number', function () {
+                expect(sqrt(4)).to.eql(2);
+                expect(sqrt(-1)).to.eql(NaN);
+                expect(sqrt(4.2)).to.eql(2.04939015319192);
+                expect(sqrt('asdf')).to.eql(NaN);
+            });
+
+            it('should return sqrt round number to precision', function () {
+                expect(_.round.apply(_, [sqrt(4.2), 3])).to.eql(2.049);
+                expect(_.round.apply(_, [sqrt(10), 4])).to.eql(3.1623);
+                expect(_.round.apply(_, [sqrt(-10), 3])).to.eql(NaN);
+            });
+        });
+
+        describe('subtract', function () {
+
+            var params;
+
+            beforeEach(function () {
+                params = datasets.subtract();
+            });
+
+            it('should return subtract two numbers', function () {
+                expect(params).to.have.length(2);
+                expect(_.subtract.apply(_, params)).to.eql(15);
+                expect(_.divide.apply(_, params)).to.eql(Infinity);
```

Comment:
To be honest I am not against it, but you would have to ask @blabno If this is acceptable. The thing is that person may not know any of previously explained functions (i.e. divide, multiply, add or whatever) and understand this one, however to solve this one you need to know all of them.